mash_client/cli.py

```diff
@@ -159,42 +160,51 @@ def delete(context, job_id):
 @click.group()
 def account():
     """
-    Handle mash account requests.
+    Submit account requests to the MASH server.
     """
 
 
-@click.command(name='delete')
+@click.command(
+    name='delete', context_settings=dict(token_normalize_func=str.lower)
+)
 @click.option(
     '--delete',
     is_flag=True,
     callback=abort_if_false,
     expose_value=False,
+    help='Force deletion without prompt.',
```

Comment:
? I don't understand this at all.

mash account delete --delete

right?

But the help says "Force deletion without prompt" but there is still a prompt?

If we want a no prompt option then that should be "--force"

Reply:
--delete shouldn't be exposed. It provides an automatic callback to confirm the deletion. So when you delete:

```
> mash account delete ...
Are you sure you want to delete account [y/n]?
```

--delete can be passed in but it's really a hidden flag.

Reply:
But it could be --force.